terraform/environments/nomis/modules/nomis_stack/variables.tf

```diff
@@ -132,4 +132,40 @@ variable "weblogic_instance_type" {
   type        = string
   description = "ec2 instance type to use for the weblogic instance"
   default     = "t2.medium"
-}
\ No newline at end of file
+}
+
+# Variables for weblogic configuration
+variable "DB_NAME" {
+  type        = string
+  description = "Name of Nomis database e.g. CNOMT1"
+}
+
+variable "DB_PORT" {
+  type        = string
+  description = "Nomis database port e.g. 1521"
```

Comment:
maybe put a default value in here, save the number of inputs we need to add to application_variables.json?